doc/depthFirstSearch/pgr_depthFirstSearch.rst

```diff
@@ -59,27 +58,34 @@ Signatures
 
 .. code-block:: none
 
-    pgr_depthFirstSearch(edges_sql, start_vid [, max_depth] [, directed])
-    pgr_depthFirstSearch(edges_sql, start_vids [, max_depth] [, directed])
+    pgr_depthFirstSearch(Edges SQL, Root vid [, directed] [, max_depth])
+    pgr_depthFirstSearch(Edges SQL, Root vids [, directed] [, max_depth])
 
-    RETURNS SET OF (seq, depth, start_vid, node, edge, cost, agg_cost)
+    RETURNS SET OF (seq, depth, Root vid, node, edge, cost, agg_cost)
     OR EMPTY SET
 
 .. rubric:: Using defaults
 
 .. code-block:: none
 
-    pgr_depthFirstSearch(TEXT edges_sql, BIGINT start_vid)
+    pgr_depthFirstSearch(TEXT, BIGINT)
 
-    RETURNS SET OF (seq, depth, start_vid, node, edge, cost, agg_cost)
+    RETURNS SET OF (seq, depth, Root vid, node, edge, cost, agg_cost)
```

Comment:
This is a column name: root_vid, only the IN compulsory parameters do not have names